tests/unit/recommenders/models/test_sar_singlenode.py

```diff
@@ -413,3 +418,41 @@ def test_match_similarity_type_from_json_file(header):
 
     # make sure fit still works when similarity type is loaded from a json file
     model.fit(train)
+
+def test_get_user_and_item_based_topk(header):
```

Comment:
minor comment, typically we try to test each function separately, so we would have `test_get_user_based_topk` and `test_get_item_based_topk`, then it is easier for us to identify which test is failing